packages/web/test/web_test.js

```diff
@@ -1,5 +1,8 @@
+import { resolve } from 'path';
+
```

Comment:
Let's remove the extra blank lines between imports.

Reply:
I intentionally added the newline, since I think it's cleaner to order (and separate) imports based on stdlib vs third-party vs local imports (we already do this for Treeherder). Ideally in the future all Neutrino imports would be sorted this way, and enforced via eslint-plugin-import.